File: .config/hiss/testdata/HISS-12/python/positive/base64-wrapped-token.py
"""HISS-12: a GitHub personal access token base64-wrapped before it was committed.
gitleaks decodes it and reports github-pat with tag decoded:base64."""

import base64

ENCODED_TOKEN = "Z2hwXzAxNkM3RDIzNDVCNkU3ODlGMDEyMzQ1Njc4OUFCQ0RFRjAxMg=="


def token() -> str:
    return base64.b64decode(ENCODED_TOKEN).decode("ascii")

File: .config/hiss/testdata/HISS-12/typescript/positive/base64-wrapped-token.ts
// HISS-12: a GitHub personal access token base64-wrapped before it was committed.
// gitleaks decodes it and reports github-pat with tag decoded:base64.
const ENCODED_TOKEN = "Z2hwXzAxNkM3RDIzNDVCNkU3ODlGMDEyMzQ1Njc4OUFCQ0RFRjAxMg==";

export function token(): string {
  return atob(ENCODED_TOKEN);
}
